CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Change Log
 
+## 2.6.0 - 2020-06-10
+- Added calls to X-Ray daemon for rules/targets with a simple JDK-based client
+[PR 145](https://github.com/aws/aws-xray-sdk-java/pull/145)
+- Added a public API for creating Emitter without exposing implementation details
+[PR 146](https://github.com/aws/aws-xray-sdk-java/pull/146)
+- Exposed sampling manifest URL for testing
+[PR 149](https://github.com/aws/aws-xray-sdk-java/pull/149)
+- Added utility executors for context propagation
+[PR 150](https://github.com/aws/aws-xray-sdk-java/pull/150)
+- Modified exception handling in UDPEmitter
+[PR 151](https://github.com/aws/aws-xray-sdk-java/issues/151)
+- Added gradle build configuration
```

Comment:
I would leave out build tooling notes from the ChangeLog they aren't really relevant to users of the library. So everything about Gradle and the various plugins and JMH version.